docs/tab-widgets/distributed-trace-receive.asciidoc

```diff
@@ -0,0 +1,165 @@
+// tag::go[]
+
+// Need help with this example
+
+1. Parse the incoming TraceContext with
+https://godoc.org/go.elastic.co/apm/module/apmhttp#ParseTraceparentHeader[`ParseTraceparentHeader`] or
+https://godoc.org/go.elastic.co/apm/module/apmhttp#ParseTracestateHeader[`ParseTracestateHeader`].
+
+2. Start a new transaction or span as a child of the incoming transaction with
+{apm-go-ref}/api.html#tracer-api-start-transaction-options[`StartTransactionOptions`] or
+{apm-go-ref}/api.html#transaction-start-span-options[`StartSpanOptions`].
+
+Example:
+
+[source,go]
+----
+// Receive incoming TraceContext
+traceContext, _ := apmhttp.ParseTraceparentHeader(r.Header.Get("Traceparent")) <1>
+traceContext.State, _ = apmhttp.ParseTracestateHeader(r.Header["Tracestate"]...) <2>
+
+opts := apm.TransactionOptions{
+	Start: time.Now(),
```

Comment:
```suggestion
```

The current time will be used if Start is unspecified. Seeing as this example is about context propagation, I'd leave this out to minimise distractions.